### DISCUSSION TITLE
Has WiFi speed improved ?

### DISCUSSION THREAD

**Discussion by bginvestor** [luc-github/ESP3D#840] [Questions]
Hi Folks,

I just stumbled on ESP8266 - ESP3D on youtube.  I want to try this.  However, reviews are saying that the file transfers via WiFi are really slow.  Has there been any improvements or new Wifi modules to speed this up?  If yes, please let me know.

I am interested in using a SKR 1.4 board.

Thanks !
Bg

**Comment by luc-github** [comment]
The issue is not the wifi, the issue is the serial file transfer using M28/M29

The file transfer status is here: https://github.com/luc-github/ESP3D/discussions/540#discussioncomment-2897564

For Serial file transfer the plan is to dev a new protocol: https://github.com/luc-github/ESP3D/issues/575

But I have others topic to be done first.
on ESP32 board an alternative solution will be to use https://github.com/luc-github/ESP3D/issues/243 which come soon because it is a key point of another project https://github.com/luc-github/ESP3D-TFT

**Comment by bginvestor** [comment]
Thank you sirSent from my iPhoneOn Dec 30, 2022, at 11:01 PM, Luc ***@***.***> wrote:﻿
The issue is not the wifi, the issue is the serial file transfer using M28/M29
The file transfer status is here: #540 (comment)
For Serial file transfer the plan is to dev a new protocol: #575
But I have others topic to be done first.
on ESP32 board an alternative solution will be to use #243 which come soon because it is a key point of another project https://github.com/luc-github/ESP3D-TFT

**Comment by liqijian101** [comment]
My research achievements:
The transmission speed of ESP8266 is 180KB/s
The transmission speed of ESP32S3 or C3 is 500KB/s, even when the high-speed tf card is used, it reaches 800KB/s.
This is my esp32S3 board：[](https://oshwhub.com/liqijian101/esp3d_esp32__camer-v2)
I've been trying to improve the transmission speed

**Reply by liqijian101** [reply]
https://oshwhub.com/liqijian101/esp3d_esp32__camer-v2
